TACC/Core-Portal
# Task/FP-1052 - Remove work2 welcome message

## Overview: ##

Remove work2 welcome message - Remove the `Work2Message.js` component and update all related files to no longer support this message.

## Related Jira tickets: ##

* [FP-1052](https://jira.tacc.utexas.edu/browse/FP-1052)

## Summary of Changes: ##

* Deleted Work2Message.js
* Deleted "import Work2Message..." from DataFiles.js (formerly line 21)
* Deleted "messages={<Work2Message />}" from DataFiles.js (formerly line 172)
* Deleted WORK2 from welcome.reducers.js

## Testing Steps: ##
This was difficult to test since /Work2 is no longer in use.
1. Used console.log statements to verify WORK2 was no longer referenced in state
2. Ran test suites - all 101 passed.

## UI Photos:

## Notes: ##
